lib/execution_engine2/db/MongoUtil.py

```diff
@@ -263,6 +264,47 @@ def check_if_already_finished(job_status):
             return True
         return False
 
+    def update_jobs_to_queued(
+        self, job_id_pairs: List[JobIdPair], scheduler_type: str = "condor"
+    ) -> None:
+        """
+        Updates a list of created jobs to queued. Does not work on jobs that already have gone through a status transition
+        :param job_id_pairs: A list of pairs of Job Ids and Scheduler Ids
+        :param scheduler_type: The scheduler this job was queued in, default condor
+        """
+        bulk_operations = []
+        now = time.time()
+        for job_id_pair in job_id_pairs:
+            if None in job_id_pairs:
+                raise InvalidStatusTransitionException
+            bulk_operations.append(
+                UpdateOne(
+                    {
+                        "_id": ObjectId(job_id_pair.job_id),
+                        "status": {
+                            "$in": [Status.created.value, Status.estimating.value]
```

Comment:
I'm wondering about the estimating state here. I'm not sure how a job gets to or leaves that state so it's not clear to me whether allowing a job that still in the estimating phase to move to the queued phase makes sense.

Reply:
The lifecycle is Create->Estimating->Queued . Estimated is actually queued in condor, but its a second job.. We could remove it for now. In the future,  the function can be renamed to "update_jobs_to_queued_or_estimating" and allow for updating to estimating here, or we can create a separate function instead.

Reply:
My preference would be to remove it until we actually implement estimation

Reply:
done

Reply:
:+1: